app/mindfulness/page.tsx

```diff
@@ -75,55 +14,27 @@ function Home() {
       </div>
 
       <div style={{ marginLeft: '20px', marginRight: '20px' }}>
-        <h2 style={{ fontSize: '30px', fontWeight: 'bold', display: 'inline-block' }}>Affirmation</h2>
-        <button
-          style={{ border: 'none', display: 'inline-block', fontSize: '24px', padding: '10px' }}
-          onClick={addButton}
-        >
-          <svg xmlns="http://www.w3.org/2000/svg" width="10%" height="10%" viewBox="0 0 100 100">
-            <circle cx="50" cy="50" r="48" fill="#f7f3f3" stroke="none" />
-            <line x1="30" y1="50" x2="70" y2="50" stroke="#d68071" strokeWidth="8" />
-            <line x1="50" y1="30" x2="50" y2="70" stroke="#d68071" strokeWidth="8" />
-          </svg>
-        </button>
-      </div>
-
-      {buttons.map((button) => (
-        <div key={button.id} className="dropdown">
-          <button className="dots-button"> {button.text}</button>
-          <div className="dropdown-content">
-            <button style={{ border: '0.5px solid black' }} onClick={() => saveAffirmation(button.text)}>
-              Apply
-            </button>
-            <button style={{ border: '0.5px solid black' }} onClick={() => openModal(button)}>
-              Edit
-            </button>
-            <button style={{ border: '0.5px solid black' }} onClick={() => deleteButton(button.id)}>
-              Delete
-            </button>
+        <h2 style={{ fontSize: '30px', fontWeight: 'bold', display: 'inline-block' }}>Mindfulness</h2>
+          <div className="flex flex-col items-center mt-4">
+            <div style={{ position: 'relative', width: '100%', height: 0, paddingBottom: '56.25%', marginBottom: '20px' }}>
+              <iframe
+                className='video'
+                title='Youtube player'
+                style={{ position: 'absolute', top: 0, left: 0, width: '100%', height: '100%' }}
+                src={`https://www.youtube.com/embed/${youtube1ID}`}
+              ></iframe>
+            </div>
+
+            <div style={{ position: 'relative', width: '100%', height: 0, paddingBottom: '56.25%' }}>
```

Comment:
Rather than repeating the same code twice, can we iterate over a list of video ids?

In addition to saving (a tiny bit of) code, this also makes it easier to see that the two videos are two of "the same thing".

Reply:
solved

Reply:
I am also still seeing two videos showed separately with two IDs rather than iteration over a list of video ids as Ben suggested. Do we have the latest code here for this PR?

Reply:
Let's push this to the next PR, I'll comment there.